src/annotator/plugin/pdf.js

```diff
@@ -203,4 +223,49 @@ export default class PDF {
 
     refreshAnnotations.map(annotation => this.annotator.anchor(annotation));
   }
+
+  /**
+   * Return the scrollable element which contains the document content.
+   *
+   * @return {HTMLElement}
+   */
+  contentContainer() {
+    return /** @type {HTMLElement} */ (document.querySelector(
+      '#viewerContainer'
```

Comment:
Maybe. At the moment the only PDF-mode thing that is not inherently coupled to PDF.js is the decision on whether side-by-side mode should be used. In case we made it user-configurable it would be up to higher level code to handle that.